css/style.css

```diff
@@ -23,3 +23,53 @@
   margin-top: 100px;
 }
 
+/* Andrea's styles below */
```

Comment:
Andrea -  You will need to set up a sep css file if you want to make changes to the CSS - label it css/andreastyles.css and then put your style changes in there - also make sure you add a link in your blog post head section so that the file knows to look for your styles in there.